drivers/net/mlx5/mlx5_prm.h

```diff
@@ -1275,4 +1276,37 @@ struct mlx5_ifc_create_mkey_in_bits {
 	struct mlx5_ifc_klm_bits klm_pas_mtt[0];
 };
 
+struct mlx5_ifc_tisc_bits {
+	u8         strict_lag_tx_port_affinity[0x1];
+	u8         reserved_at_1[0x3];
+	u8         lag_tx_port_affinity[0x04];
+	u8         reserved_at_8[0x4];
+	u8         prio[0x4];
+	u8         reserved_at_10[0x10];
+	u8         reserved_at_20[0x100];
+	u8         reserved_at_120[0x8];
+	u8         transport_domain[0x18];
+	u8         reserved_at_140[0x8];
+	u8         underlay_qpn[0x18];
+	u8         reserved_at_160[0x3a0];
+};
+
+struct mlx5_ifc_create_tis_out_bits {
+	u8         status[0x8];
+	u8         reserved_at_8[0x18];
+	u8         syndrome[0x20];
+	u8         reserved_at_40[0x8];
+	u8         tisn[0x18];
+	u8         reserved_at_60[0x20];
+};
+
+struct mlx5_ifc_create_tis_in_bits {
+	u8         opcode[0x10];
+	u8         uid[0x10];
```

Comment:
Like we did in all other commands Input structures, let's put reserved instead of uid (as it's not exposed to user)

Reply:
changed it to reserved_at_10